token/js/src/extensions/groupMember/state.ts

```diff
@@ -0,0 +1,30 @@
+import { struct, u32 } from '@solana/buffer-layout';
+import { publicKey } from '@solana/buffer-layout-utils';
+import { PublicKey } from '@solana/web3.js';
+import type { TokenGroupMember } from '@solana/spl-token-group';
+import type { Mint } from '../../state/mint.js';
+import { ExtensionType, getExtensionData } from '../extensionType.js';
+
+/** Buffer layout for de/serializing a TokenGroupMember extension */
+export const TokenGroupMemberLayout = struct<TokenGroupMember>([
+    publicKey('mint'),
+    publicKey('group'),
+    u32('memberNumber'),
+]);
```

Comment:
Same with this layout!
https://github.com/solana-labs/solana-program-library/blob/7e022ac1b2ac527d6607712521685e64d0ff50f8/token-group/js/src/state/tokenGroupMember.ts#L30-L37